htdocs/PI/index.php

```diff
@@ -414,6 +413,13 @@ function authByCert() {
         }
     }
 
+    function authByAnyIdentifier()
+    {
+        if (empty($this->identifier)) {
+            die("No valid identifier found. Try accessing the " .
```

Comment:
Why do we die rather than throw an exception as in authByIdentifier()?

Reply:
not sure, it's "always" been like that. Given `getXml()` is one giant `try... catch` throwing an exception here seems better.

I think all the other methods in `PI/index.php` now throw an exception, so I might also tweak the logic in the [catch](https://github.com/GOCDB/gocdb/blob/dev/htdocs/PI/index.php#L364) slightly

Reply:
See 765e9f0601cb554b0495538ddff44c49c18c59a0